pkg/grizzly/config.go

```diff
@@ -11,6 +11,10 @@ type Opts struct {
 	Directory    bool
 	JsonnetPaths []string
 	Targets      []string
+
+	// Used for supporting commands that output dashboard JSON
+	Folder string
+	Legacy bool
```

Comment:
I'm not super convinced about "legacy", but I don't have any super convincing alternative. What about something more specific, like SpecOnly? Or something like that, that states more clearly that it just manipulates the "spec" part?

In fact, I don't think this is legacy (yet, at least).